src/vaadin-overlay.html

```diff
@@ -286,12 +291,16 @@
 
           /**
            * References the content container after the template is stamped.
+           * @type {HTMLElement}
```

Comment:
This now results in:
```ts
content: HTMLElement|null;
```
but I think null is unnecessary but it should have `undefined` since that is the initial value and there's also one line where this is explicitly set to `undefined` (in `_removeOldContent()`).

Suggestion:
```suggestion
           * @type {!HTMLElement | undefined}
```
results in:
```ts
content: HTMLElement|undefined;
```

Reply:
Thanks, done.